mips/malta.c

```diff
@@ -172,7 +172,7 @@ void platform_init(int argc, char **argv, char **envp, unsigned memsize) {
 
   setup_kenv(argc, argv, envp);
 
-  uart_init();
+  cn_init();
```

Comment:
I suppose some consoles may appear later in the boot process, e.g. as result of probing available devices. Do we plan to have a mechanism for re-selecting the best available console?

Also some UART devices (e.g. the two 16550s on ISA bus) can't be probed for availability, and it's up to the platform configuration to decide whether they are present or not. Supporting them is barely a matter of picking slightly different register addresses. I though it might be cool to direct kernel messages to the "ttyCBUS", while user programs would use "tty0" (@ `0x03f8`). Though that would require multiple independent consoles, and a way of opening a particular one...